dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1594,6 +1662,6 @@ def np_type_to_type(val):
         """
         if isinstance(val, np.integer):
             return int(val)
-        if isinstance(val, np.float):
```

Comment:
Is this intentional to remove?
what if instead we did `np.floating`? would handling float32 mess up the input as well?

Reply:
`np.float` is a deprecated alias for `float` since [NumPy 1.20.0](https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations). I think using `np.floating` is a good idea.